MapleServer2/Packets/ResponseLoadUGCMapPacket.cs

```diff
@@ -0,0 +1,51 @@
+﻿using MaplePacketLib2.Tools;
+using MapleServer2.Constants;
+using MapleServer2.Enums;
+using MapleServer2.Types;
+
+namespace MapleServer2.Packets
+{
+    public class ResponseLoadUGCMapPacket
+    {
+        public static Packet LoadUGCMap(bool isHome, Home home = null)
+        {
+            PacketWriter pWriter = PacketWriter.Of(SendOp.LOAD_UGC_MAP);
+            pWriter.WriteLong();
+            pWriter.WriteBool(isHome);
+
+            if (isHome)
+            {
+                pWriter.WriteLong(home.AccountId);
+                pWriter.WriteUnicodeString(home.Name);
+                pWriter.WriteUnicodeString(home.Description);
+                pWriter.WriteByte();
+                pWriter.WriteInt(home.ArchitectScoreCurrent);
+                pWriter.WriteInt(home.ArchitectScoreTotal);
+                pWriter.WriteInt();
+                pWriter.WriteInt();
+                pWriter.WriteByte();
+                pWriter.WriteByte(home.Size);
+                pWriter.WriteByte(home.Height);
+                pWriter.WriteByte(home.Background);
+                pWriter.WriteByte(home.Lighting);
+                pWriter.WriteByte(home.Camera);
+                pWriter.WriteByte(9);
```

Comment:
Does `home.Permissions.Length/Size/Count` not work here?

Reply:
Oh yes I could do that, oops!

Reply:
No I can't, permissions don't always have all entries!

Reply:
Oh yeah true. Any way for 9 to not be a magic number? Perhaps a static constant inside the `Home` file?

Reply:
Unrelated, but what about ripping out lines 18 to 44 into a separate method called `WriteHome(home)`, I feel like this probably isn't the only time we'll need this, and it looks nicer. Optional though